modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp

```diff
@@ -317,6 +317,25 @@ ItemSelectorRepositoryEntryManagementToolbarDisplayContext itemSelectorRepositor
 
 									Map<String, Object> data = new HashMap<>();
 
+									Map<String, Serializable> customFieldsMap = fileEntry.getExpandoBridge().getAttributes();
```

Comment:
Here, unless there's a very good reason, you should serialize the expandos using the regular expando format, and not a custom JSON.